docs/src/routes/docs/components/star-rating.svx

```diff
@@ -0,0 +1,91 @@
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+
+  let restaurantRating = 1;
+</script>
+
+# StarRating
+
+An input element used for rating with a number of stars.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <StarRating name="restaurant" bind:value={restaurantRating} />
+    <br>
+    <StarRating name="hotel" value={5} max={7} disabled>
+      <span slot="icon"><ThumbsUpIcon size="24" /></span>
+    </StarRating>
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  let restaurantRating = 1;
+</script>
+
+<StarRating name="restaurant" bind:value={restaurantRating} />
+<br>
+<StarRating name="hotel" value={5} max={7} disabled>
+  <span slot="icon"><ThumbsUpIcon size="24" /></span>
+</StarRating>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| ==**`name`**==&nbsp;<span class="required">*</span> |  | `string` | The name to assign to all stars belonging to the same group. Check [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#htmlattrdefname) for more information. |
+| **`max`** | `5` | `number` | How many stars to be displayed. |
+| **`value`** | `null` | `number` | Current amount of selected stars. |
+| **`disabled`** | `false` | `boolean` | Disables star selection. |
+| **`events`** | `[]` | `Array` | List of handlers for the [Events action](): `[{ name, handler }]`. |
+| **`...`** |  | `any` | Everything else will be passed to the underlying `<input type="radio">` element. |
+
+<div class="required">* Required</div>
+
+### Class Properties
+
+All `<StarRating>` components use the `.star-rating` class.
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the component. |
+| **`starClass`** | `null` | `string` | A class string to assign to label element containing the star icon. |
+
+
+## Events {#events}
+
+| Name | Event Detail | Description |
+| ---- | ------------ | ----------- |
+| **`change`** | `{ value, nativeEvent }` | Fired when the selection changes. The `value` contains the number of selected stars and `nativeEvent` has the [native change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event). |
+
+
+## Slots {#slots}
+
+### `icon` slot
+
+The icon to use as the star. Defaults to [Feather icons](https://feathericons.com/?query=star)' `Star`.
+
+
+## SCSS Variables {#scss-variables}
+
+| Name | Description | Default |
+| ---- | ----------- | ------- |
+| **`$main`** | Affects the hover and background colors of the component and the default values of `$hover-color` and `$selected-color`. | <ColorPreview value="#4300B0" /> |
+| **`$hover-color`** | The background color of the label when hovering on it with the mouse. | `mix(white, $main, 60%)` |
```

Comment:
This one and the next are not customizable, so I see no reason to list them here. Besides, their values are derived from `$main`. Instead, I suggest to update the `$main` description to incorporate both of those descriptions in some brief way